pom.xml

```diff
@@ -20,11 +20,27 @@
     </properties>
 
     <modules>
+        <module>service</module>
     </modules>
 
     <groupId>dk.dbc</groupId>
     <artifactId>ess-payara</artifactId>
     <version>1.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <configuration>
+                        <source>1.8</source>
+                        <target>1.8</target>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+
 </project>
```

Comment:
Man kan muligvis blive fri fra Javadoc advarsler for genereret kode
(https://is.dbc.dk/job/ess-payara/job/SE-2769/8/analysis/)
ved at tilføje:
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <version>2.9.1</version>
                <artifactId>maven-javadoc-plugin</artifactId>
                <configuration>
                    <excludePackageNames>dk.dbc.openformat,dk.dbc.sru</excludePackageNames>
                </configuration>
            </plugin>
        </plugins>

Reply:
Kiggede lige igen på javadoc warnings for https://is.dbc.dk/job/ess-payara/job/master/21/

Konfigurations af maven-javadoc-plugin skal stå i projekt rod pom.xml, fordi det bliver kørt med "mvn javadoc:aggregate" i rod folderen

excludePackageNames skal være "dk.dbc.openformat,dk.dbc.sru.*", fordi openformat klassene ligger i dk.dbc.openformat pakken, men dk.dbc.sru klasserne ligger i subdirectories *